tests/helpers/test_device_registry.py

```diff
@@ -42,13 +42,15 @@ async def test_get_or_create_returns_same_entry(hass, registry, update_events):
         name="name",
         manufacturer="manufacturer",
         model="model",
+        suggested_area="Man Cave",
```

Comment:
I recommend using another test name. Eg "Game Room".

Reply:
Good idea. That made my day ❤️